### PR TITLE
Reballances the disabler damage and uncuff/unbuckle time

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

* Reduces disabler damage (eguns use a little less energy for firing disabler shots)
* Reduces the time one needs to uncuff/unbuckle with cuffs.

## Why It's Good For The Game

Roundstart sec gear is ridiculously overpowered for what it does, and continues to be powerful throughout the game. 

Disablers have large ammo capacity and deal large amounts of stam damage. They are roundstart available for every officer and very powerful for what they do. There is very little reason to replace them.

Cuffs/Buckles are effectively a 2 minute stun/inability to use your hands that is really easy to pull off and hard to escape. This makes it more risky, while still as effective, but not excessively effective. 

## Changelog
:cl:
balance: Reduced the efficency of disablers
balance: You can unbuckle faster from cuffs and beds
/:cl:

### CODE REVIEW

**Comment by bloons3** [comment]
Cuffs are meant to be hard to get out of for a reason, same thing with cable cuffs.

There's a reason Beepsky cuffs you with zip-cuffs instead of actual handcuffs...

**Comment by Archanial** [comment]
the madman

**Comment by ToasterBan** [comment]
do you want security to just ignore all minor crimes and just validhunt? because this is what will happen if you keep on nerfing security because non-lethals would be so useless your only job would be to kill valids

**Comment by PowerfulBacon** [comment]
Espee sec rebalance round 2

File: code/modules/projectiles/ammunition/energy/stun.dm [hunk 1]
```diff
@@ -25,5 +25,8 @@
 	fire_sound = 'sound/weapons/taser2.ogg'
 	harmful = FALSE
 
+/obj/item/ammo_casing/energy/disabler/egun
+	e_cost = 20
```
Comment (PowerfulBacon) [inline code/modules/projectiles/ammunition/energy/stun.dm:29]: Holy shit

**Comment by Archanial** [comment]
I think I've seen it somewhere
#3935

**Comment by eeSPee** [comment]
I have attained 8 thumbs down.

Speedmerge when?

**Comment by ToasterBan** [comment]
having handcuffs mean you lost the battle, why should you have a chance to fight back once you lose?

**Comment by PowerfulBacon** [comment]
The handcuff stuff isnt bad, the timers for buckle cuffing are overly long anyway, but the stamina damage changes and ammo changes are stupid. That's like 50 shots of disabler bullets

**Pull request closed without merging**